src/commands/run.rs

```diff
@@ -224,6 +224,13 @@ pub struct RunCommand {
     #[clap(long)]
     trap_on_grow_failure: bool,
 
+    /// Enables memory error checking by including wmemcheck_state in
```

Comment:
No need to refer to the implementation ("including wmemcheck_state") here -- the relevant bit for the CLI user is just that it does memory-usage error checking.